src/aws/nodeworker/nodeworker.py

```diff
@@ -47,20 +51,38 @@ async def heartbeat(self):
     async def process(self):
         """
         Start function for the WorkerCore.
+
+        return:
+            np.array
+                Binary 1x6 Numpy array containing the results of the prediction i.e. [0,1,1,0,1,1]
         """
         try:
             while True:
                 if not self.current_task and self._task_queue:
                     self.current_task = self._task_queue.pop(0)
+
                     log_info("Downloading File " + self.current_task.file_path + ".")
                     file = self.storage_connector.download_file(
                         file_path=self.current_task.file_path,
                         key=self.current_task.key
                     )
                     log_info("Downloaded file " + self.current_task.file_path + ".")
+                    # file = self.storage_connector.download_file(
+                    #     file_path=self.current_task.file_path,
+                    #     key=self.current_task.key
+                    # )
```

Comment:
The above needs to be introduced again for the actual files that should be processed. The task file path needs to be added to the command packet (and should come from the Input.csv), it should then be uploaded in the node manager. The key used to upload this file should also be added to the command packet. Then the task is sent to the worker. The worker can then download the file with the given file path and key and start processing.